OJP/OJP_Trips.xsd

```diff
@@ -813,12 +860,22 @@
 			</xs:element>
 			<xs:element name="Count" type="xs:positiveInteger" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Number how often the access feature occurs in this PathLink</xs:documentation>
+					<xs:documentation>Number indicating how often the access feature occurs in this PathLink</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="AccessFeatureStatus" type="AccessFeatureStatusEnumeration" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Whether the access feature is available or out of service.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="AccessFeatureStatusText" type="InternationalTextStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Textual information about reduced availability of the access feature, in particular if AccessFeatureStatus is partiallyAvailable.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="AccessibilityFeature" type="AccessibilityFeatureTypesEnumeration" minOccurs="0" maxOccurs="unbounded">
```

Comment:
Why should this be removed? In my understanding, this is the main means to return more detailed accessibility information (more detailed than only the access equipment on the PathLink).

Reply:
I guess this was a problem with the rebasing from my side.

Reply:
@trurlurl  should work now again.